dcap/types/src/quote3.rs

```diff
@@ -65,20 +74,34 @@ impl Error {
     }
 }
 
+impl From<nom::Err<nom::error::Error<&[u8]>>> for Error {
+    fn from(_: nom::Err<nom::error::Error<&[u8]>>) -> Self {
+        // nom errors rely on the lifetime of the data being parsed, so we
+        // convert.
+        // nom errors should only happen if the size was wrong for decoding the
+        // quote.
+        Error::InputLength {
+            required: MIN_QUOTE_SIZE,
+            actual: 0,
```

Comment:
Not fond of this actual size being 0 here.
Also not fond of that this relies on all nom usages to have  preguard ensuring nom shouldn't fail.
Up for suggestions here.  Could change nom usages to `except()` statements, but errors seem like a nicer behavior for customers.

Reply:
Changed moving discusion to comment on new code